<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cosys</class>
 <widget class="QMainWindow" name="cosys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>cosys</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelCmd">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Command:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelRes">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Result:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="boxCmd">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="boxRes">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="barPrg">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>400</y>
      <width>721</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnExe">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineCmd">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>511</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Enter Command</string>
    </property>
   </widget>
   <zorder>labelRes</zorder>
   <zorder>boxCmd</zorder>
   <zorder>boxRes</zorder>
   <zorder>barPrg</zorder>
   <zorder>BtnExe</zorder>
   <zorder>lineCmd</zorder>
   <zorder>splitter_5</zorder>
   <zorder>labelCmd</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
